code/switch.js

```diff
@@ -0,0 +1,227 @@
+const menu = [
+    {
+        "type": "Pizza",
+        "subtype": "Margarita",
+        "cost": 150,
+        "childcost": 100
+    },
+    {
+        "type": "Pizza",
+        "subtype": "Ham & Pineapple",
+        "cost": 250,
+        "childcost": 120
+    },
+    {
+        "type": "Pizza",
+        "subtype": "Vegan Delight",
+        "cost": 300,
+        "childcost": 150
+    },
+    {
+        "type": "Pasta",
+        "subtype": "Spaghetti Bolognese",
+        "cost": 150,
+        "childcost": 100
+    },
+    {
+        "type": "Pasta",
+        "subtype": "Gnocchi",
+        "cost": 200,
+        "childcost": 150
+    },
+    {
+        "type": "Pasta",
+        "subtype": "Penne and Parmesan",
+        "cost": 100,
+        "childcost": 50
+
+    },
+    {
+        "type": "Salad",
+        "subtype": "Ceasar salad",
+        "cost": 150,
+        "childcost": 100
+
+    },
+    {
+        "type": "Salad",
+        "subtype": "Greek salad",
+        "cost": 150,
+        "childcost": 50
+    },
+    {
+        "type": "Salad",
+        "subtype": "Watermelon salad",
+        "cost": 150,
+        "childcost": 120
+    }
+];
+
+let orderType = [];
+let order = [];
+let ordertemp = [];
+let orderjoin;
+
+
+
+function ageCheck() {
+    let age = parseInt(prompt("How old are you?"));
+
+    if (age <= 0 || age > 124) {
+        alert("invalid");
+        ageCheck();
+    }
+
+    else if (age <= 5) {
+        alert(age + " years? better get your parents to order, you are way to young to order food on you own!");
+        ageCheck();
+    }
+    else if (age <= 15) {
+        order.push(orderType[0].childcost);
+        alert(age + " years? \n I guess you'll need a childrens portion. \n The total amount will be " + order[1] + " ByteCoin");
+        checkOut();
+    }
+    else if (age >= 16) {
+        order.push(orderType[0].cost);
+        alert("Seems like you are old enough for a regular portion.\n The cost is " + order[1] + " ByteCoin");
+        checkOut();
+    }
+    else {
+        alert("invalid");
+        ageCheck();
+    }
+}
```

Comment:
this checks are very good, would have been nice to see it in action on your deployed version! 👍

Reply:
hmm.. did you not see the provided links?
https://pizzabytez.netlify.app/
https://pizzabytez.netlify.app/switch

also after completion, you should see a button who will take you to the other example.

Reply:
switched the scripts so now the switch versjon loads first.